config/ember-try.js

```diff
@@ -10,6 +9,37 @@ module.exports = function() {
     return {
       useYarn: true,
       scenarios: [
+        {
+          name: 'ember-lts-2.4',
```

Comment:
I am positive we want to drop 2.4 from our support matrix ;)

Reply:
Temporary change, gone now.